app/src/main/java/com/test/lolinformation/ui/tournament/TournamentPagerAdapter.kt

```diff
@@ -0,0 +1,26 @@
+package com.test.lolinformation.ui.tournament
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import com.test.lolinformation.ui.tournament.series.SerieFragment
+
+class TournamentPagerAdapter(
+    fragmentManager: FragmentManager,
+    private val listItem: List<String>
```

Comment:
đặt tên rõ ràng vào anh 
```suggestion
    private val pageTitles: List<String>
```